rhc-ose-ansible/roles/registry/tasks/auth_layer_nginx.yaml

```diff
@@ -0,0 +1,29 @@
+---
+- name: Install Nexus Repository
+  yum:
+    name: "{{ nexus_repo_url }}"
```

Comment:
also, i think I don't want to assume i'm using the default there, as a user should be able to override most of this? What I probably want to do is add all of these to the gather facts block, right?